### PR TITLE
Branch storage corrupted exception

### LINKED ISSUE #361: Handle StorageCorruptedException
Ask if user wants to reset, or carry out troubleshooting steps by themselves.

### PR DESCRIPTION
**Pull request by durianpancakes**
Resolves #361

### CODE REVIEW

File: src/main/java/seedu/duke/storage/StorageManager.java [hunk 1]
```diff
@@ -21,6 +21,20 @@ public StorageManager(String fileName) {
     }
 
     //@@author durianpancakes
+
```
Comment (AndreWongZH) [inline src/main/java/seedu/duke/storage/StorageManager.java]: Redundant spacing here.

File: src/test/java/seedu/duke/StorageTest.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
+package seedu.duke;
+
+import org.junit.jupiter.api.Test;
+import seedu.duke.exception.StorageCorruptedException;
+import seedu.duke.storage.event.EventStorageManager;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
```
Comment (AndreWongZH) [inline src/test/java/seedu/duke/StorageTest.java:8]: Considering adding authorship here

**Review by AndreWongZH — COMMENTED** [review]
Looks goods. Just a few minor changes

File: src/main/java/seedu/duke/storage/StorageManager.java (hunk 1 above)
Comment (durianpancakes) [reply src/main/java/seedu/duke/storage/StorageManager.java]: Will do. Thank you!

File: src/test/java/seedu/duke/StorageTest.java (hunk 2 above)
Comment (durianpancakes) [reply src/test/java/seedu/duke/StorageTest.java:8]: Noted 👍

**Pull request merged**